### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ilyssaevans**
Please Review this, issabobissa

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 fun sunday demo
 
 Is GitHub fun?
+
+Yes it is
```
Comment (issabobissa) [inline README.md:6]: I agree with this line! I will explicitly approve above.

Comment (ilyssaevans) [reply README.md:6]: Sounds good. Looking out for it!

**Review by issabobissa — APPROVED** [review]
I approve.

**Comment by issabobissa** [comment]
Will you merge this pull request now, @ilyssaevans?

**Pull request merged**